backend/vkt/src/main/java/fi/oph/vkt/repository/PersonRepository.java

```diff
@@ -1,11 +1,25 @@
 package fi.oph.vkt.repository;
 
 import fi.oph.vkt.model.Person;
+import java.time.LocalDateTime;
+import java.util.List;
 import java.util.Optional;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface PersonRepository extends BaseRepository<Person> {
   Optional<Person> findByIdentityNumber(final String identityNumber);
   Optional<Person> findByOtherIdentifier(final String otherIdentifier);
+
+  @Query(
+    "SELECT p" +
+    " FROM Person p" +
+    " LEFT JOIN p.enrollments e" +
+    " LEFT JOIN p.reservations r" +
+    " WHERE p.latestIdentifiedAt < ?1" +
+    " GROUP BY p" +
+    " HAVING COUNT(e) = 0 AND COUNT(r) = 0"
+  )
```

Comment:
Ei suurta merktistä kun dataa ei varmaan kovin paljoa tule olemaan mut tän vois ehkä mielummin kirjoittaa jotenkin näin (yleensä kannattaa suosia exists kuin count kun count joutuu rullaan koko taulun läpi).

```sql
     SELECT p
     FROM Person p
     WHERE p.latestIdentifiedAt < ?1 
     AND NOT EXISTS (
         select 1 from enrollment e where p.person_id = e.person_id
     ) 
     AND NOT EXISTS (select 1 from reservation r where p.person_id = r.person_id);
```